src/test/ui/feature-gates/feature-gate-rustc_const_unstable.rs

```diff
@@ -1,9 +0,0 @@
-// Test internal const fn feature gate.
-
-#![feature(const_fn)]
-
-#[rustc_const_unstable(feature="fzzzzzt")] //~ stability attributes may not be used outside
```

Comment:
This test should still exist, just without the feature gate, right?

Reply:
Good point. I just saw "feature gate" and thought this would be for the const_fn gate.